numpy/linalg/tests/test_linalg.py

```diff
@@ -2315,6 +2315,56 @@ def test_cross():
         np.linalg.cross(x_2dim, x_2dim)
 
 
+class TestCross2d:
+
+    def test_basic_cross2d(self):
+        x = np.arange(4).reshape((2, 2))
+        actual = np.linalg.cross2d(x, x + 1)
+        expected = np.array([-1, -1])
+
+        assert_equal(actual, expected)
+
+    def test_2x2_anticommutativity(self):
```

Comment:
[It looks like NumPy already uses](https://github.com/numpy/numpy/blob/86c7f4b3dbdab809ee03b711e62f22206f95b1d7/requirements/test_requirements.txt#L6) [Hypothesis](https://hypothesis.works) for testing, and this would be a decent place for it.

I would suggest [insisting](https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.floats) the vector components be less than the square root of half the maximum representable value, or allowing the results to both be NaN if they are not negatives of each other.

I'm not sure if `assert_equal(la.cross2d(u, u), 0)` should be its own check or not.

Reply:
Haven't touched Hypothesis, and I probably don't have the bandwidth to pick that up right now.  I would happily take a suggested fix, and add you in a a coauthor if you want to clean up the test accordingly.  In this section, I modified the cross (3d) stuff again, and then added a few other things that I noticed were causing problems.

Reply:
I opened a PR as bmwoodruff/numpy#1, or you could accept the suggestion in the GitHub file review interface and update the imports yourself.

Reply:
I've been waiting to hear the status on whether or not we'll be adding a cross2d to the code base.  I saw the PR on my repo, but haven't merged it in yet because I haven't taken the time to review it and know what it is doing. If/when it's decided to add in a cross2d, then I'll make the bandwidth to review the PR and merge it in. 

For now, I'm waiting. If there is time at the next triage meeting, I hope to discuss this and merge things.  Perhaps the suggested one-liner by @seberg is enough to remove any need for a cross2d.

Maybe the right direction to head is to get feedback from those using the deprecated 2D version on whether or not the suggested 1 line change is sufficient.

Reply:
That sounds sensible